docs/_howto/stub-dependency.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: How to stub a dependency of a module?
+---
+
+To stub dependency(imported module) of a module under test you have to import it explicitly in your test and stub the desired method.
```

Comment:
missing a space.

Reply:
You might add that Sinon is simply a stubbing library, not a module intercepting library, and how you stub out dependencies is highly dependant on your environment. For Node environments, we usually recommend solutions targetting [link seams](https://github.com/sinonjs/sinon/blob/8683af388a2b46b2b3e7764fe6fcfe91b8543e35/docs/_howto/link-seams-commonjs.md) (replace with relative link) or explicit dependency injection. Though in _some simple cases_, you can get away with just using Sinon by modifying the module exports of the dependency. 

(_And then comes your text_)